src/EPR.Calculator.API/Controllers/CalculatorInternalController.cs

```diff
@@ -146,6 +150,7 @@ public IActionResult UpdateRpdStatus([FromBody] UpdateRpdStatus request)
         [Route("prepareCalcResults")]
         public IActionResult PrepareCalcResults([FromBody] CalcResultsRequestDto resultsRequestDto)
         {
+            this.transposePomAndOrgDataService.Transpose(resultsRequestDto.RunId);
```

Comment:
Can you please pass whole of CalcResultsRequestDto instead of RunId.
It will be advantage later so that it will not be a breaking change.

Reply:
I have updated the code as per your suggestion.